src/main/java/org/folio/edge/oaipmh/OaiPmhHandler.java

```diff
@@ -132,6 +138,7 @@ protected void handleProxyResponse(RoutingContext ctx, HttpResponse<Buffer> oaiP
     } else {
       var message = String.format(ERROR_FROM_REPOSITORY, oaiPmhResponse.statusCode(), oaiPmhResponse.statusMessage() + " " + oaiPmhResponse.bodyAsString());
       log.error(message);
+      log.debug("Response ended status: {}", ctx.response().ended());
```

Comment:
Why do we need debug information here? this else-section corresponds error handling.